setup.py

```diff
@@ -59,8 +59,8 @@
                         'python-dateutil',
                         'packaging',
                         'psutil>=3.3.0',
-                        'intervaltree<3.0', # See Gallopsled/pwntools#1238
-                        'sortedcontainers<2.0', # See Gallopsled/pwntools#1154
+                        'intervaltree',
```

Comment:
Actually, would be nice to explicitly state that old intervaltree won't work.
```suggestion
                        'intervaltree>=3.0',
```